ee/codegen_integration/fixtures/simple_guardrail_node/code/display/nodes/__init__.py

```diff
@@ -1,4 +1,4 @@
 from .final_output import FinalOutputDisplay
 from .guardrail_node import GuardrailNodeDisplay
 
-__all__ = ["GuardrailNodeDisplay", "FinalOutputDisplay"]
+__all__ = ["FinalOutputDisplay", "GuardrailNodeDisplay"]
```

Comment:
[nit] let's add the config that always add the comma after each element so that it's always consistently an item per line